### PR TITLE
Add support for workgroup

### PR DESCRIPTION
**Pull request by WarFox**
## Description

Add support for configuring Athena Workgroup.

https://docs.aws.amazon.com/athena/latest/ug/workgroups.html

## Checklist

- [x] I've added this contribution to the `changelog.md`.
- [x] I've added my name to the `AUTHORS` file (or it's already there).

### CODE REVIEW

**Comment by zzl0** [comment]
@WarFox 👍  Thanks for your PR, I will take a look tomorrow.

File: athenacli/athenaclirc [hunk 1]
```diff
@@ -14,6 +14,9 @@ role_arn = '' # e.g. arn:aws:iam::987654321000:role/MyAthenaAccessRole
 # The format is 's3://<your s3 directory path>'
 s3_staging_dir = ''
 
+# Athena workgroup used to run the query
+work_group = 'primary'
```
Comment (zzl0) [inline athenacli/athenaclirc:18]: I think this setting will override `AWS_ATHENA_WORK_GROUP` environment variable. Sometimes people don't want to use the `athenaclirc` and want to use environment variables. What happens if we pass '' or `None` to `pyathena`? will `pyathena` correctly use `primary` as default value?

File: changelog.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 (Unreleased; add upcoming change notes here)
 ==============================================
 
+* Add support for configuring Athena workgroup
```
Comment (zzl0) [inline changelog.md:4]: Could you please add you name at the end? e.g.

`* Add homebrew installation support. ((Thanks: @chenrui333))`

**Review by zzl0 — COMMENTED** [review]
Thanks again for your PR. I just have some minor comments.

File: README.md [hunk 3]
```diff
@@ -57,6 +59,7 @@ $ export AWS_ACCESS_KEY_ID=YOUR_ACCESS_KEY_ID
 $ export AWS_SECRET_ACCESS_KEY=YOUR_SECRET_ACCESS_KEY
 $ export AWS_DEFAULT_REGION=us-west-2
 $ export AWS_ATHENA_S3_STAGING_DIR=s3://YOUR_S3_BUCKET/path/to/
+$ export AWS_ATHENA_WORK_GROUP=s3://YOUR_S3_BUCKET/path/to/
```
[the remark this answers is not in the document]
Comment (WarFox) [reply README.md:62]: this is a copy-paste mistake

File: athenacli/athenaclirc (hunk 1 above)
Comment (WarFox) [reply athenacli/athenaclirc:18]: yes, when no workgroup is provided Athena defaults to `primary` workgroup. You may have noticed queries run with current version of `athenacli` are all executed in the `primary` workgroup

Comment (zzl0) [reply athenacli/athenaclirc:18]: cool, then could we set  `work_group` to `''` here? so users can use `AWS_ATHENA_WORK_GROUP` to override the default value.

**Review by zzl0 — APPROVED** [review]
Looks great to me, thanks again!

**Comment by WarFox** [comment]
thank you for merging :tada:

When can we expect this to be in a release?

**Comment by zzl0** [comment]
@WarFox I just published a new version on `PyPI` : [`athenacli 1.6.0`](https://pypi.org/project/athenacli/)

**Comment by WarFox** [comment]
Thank you!

**Pull request merged**